package.json

```diff
@@ -28,13 +28,17 @@
     "server.json",
     "language/es5.json",
     "language/es6.json",
-    "language/es2015.json",
     "language/es2016.json",
     "language/es2017.json",
+    "language/rules-es5.json",
+    "language/rules-es6.json",
+    "language/rules-es2016.json",
+    "language/rules-es2017.json",
+    "language/not-es5.json",
     "language/not-es6.json",
-    "language/not-es2015.json",
     "language/not-es2016.json",
-    "language/not-es2017.json"
+    "language/not-es2017.json",
+    "language/es2015.json"
```

Comment:
Maintain the sort?

Reply:
I put 2015 at the bottom because it's an alias

Reply:
Fair.